packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -99,6 +99,113 @@ <h3 id="attachments-multipleFile" hidden>Attachments</h3>
   </div>
   <!-- FileUploader: end -->
 
+</section>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Input Multiple with Validation</h2>
+
+  <!-- FileUploader: start -->
+  <div id="example_multipleWithAccept" class="FileUploader" data-toggle="fileUploader">
+
+    <!-- FileUploaderAttachment template: start -->
+    <template data-spirit-snippet="item">
+      <li class="FileUploaderAttachment" data-spirit-populate-field="item">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#file" />
+        </svg>
+        <span class="text-truncate" data-spirit-populate-field="name"></span>
+        <button type="button" class="FileUploaderAttachment__remove" data-spirit-populate-field="button">
+          <span class="accessibility-hidden">Remove</span>
+          <svg width="24" height="24" aria-hidden="true">
+            <use xlink:href="/icons/svg/sprite.svg#close" />
+          </svg>
+        </button>
+      </li>
+    </template>
+    <!-- FileUploaderAttachment template: end -->
+
+    <!-- FileUploaderInput: start -->
+    <div class="FileUploaderInput" data-spirit-element="wrapper">
+      <label for="fileUploadMultipleWithAccept" class="FileUploaderInput__label">Label</label>
+      <input type="file" id="fileUploadMultipleWithAccept" name="attachment3" class="FileUploaderInput__input" multiple data-spirit-element="input" accept=".png,image/jpeg" />
+      <div class="FileUploaderInput__dropZone" data-spirit-element="dropZone">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#upload" />
+        </svg>
+        <label for="fileUploadMultipleWithAccept" class="FileUploaderInput__dropZoneLabel">
+          <span class="FileUploaderInput__link link-primary link-underlined">Upload your file(s)</span>
+          <span class="FileUploaderInput__dragAndDropLabel">or drag and drop here</span>
+        </label>
+        <div class="FileUploaderInput__helperText">️Only JPG and PNG images are allowed. Max size of each file is 10 MB</div>
+      </div>
+    </div>
+    <!-- FileUploaderInput: end -->
+
+    <!-- FileUploaderList: start -->
+    <h3 id="attachments-multipleWithAccept" hidden>Attachments</h3>
+    <ul class="FileUploaderList" aria-labelledby="attachments-multipleWithAccept" data-spirit-element="list"></ul>
+    <!-- FileUploaderList: end -->
+
+  </div>
+  <!-- FileUploader: end -->
+</section>
+<section class="docs-Section">
+
+  <h2 class="docs-Heading">Input multiple with Dismissible Feature</h2>
+
+  <p>
+    Input is hidden after the "<code>data-file-queue-limit</code>" limit is reached and "<code>data-spirit-dismissible</code>" is set.
+    After removing the file from the queue, the input will be restored.
```

Comment:
```suggestion
    After removing a file from the queue, the input will be restored.
```

This is one of the few places where the article matters quite a lot 🙂.

Reply:
fixed